android/build.gradle

```diff
@@ -64,9 +83,11 @@ allprojects {
             // react-native-background-fetch
             url("${project(':react-native-background-fetch').projectDir}/libs")
         }
-        maven {
-            // Required for react-native-background-geolocation
-            url("${project(':react-native-background-geolocation').projectDir}/libs")
+        if (project.buildWithRNBGeolocation) {
+          maven {
+              // Required for react-native-background-geolocation
+              url("${project(':react-native-background-geolocation').projectDir}/libs")
+          }
```

Comment:
Maybe this can be replaced by: 

```
if (findProject(':react-native-background-geolocation') != null) {
  maven {
    // Required for react-native-background-geolocation
    url("${project(':react-native-background-geolocation').projectDir}/libs")
  }
}
```